### ISSUE TITLE
findAllEntriesWithAllValues returns false if column name has dash character

### ISSUE THREAD

**Issue by tikyon** [jegelstaff/formulize#467]
Removing such column from `$fieldsToReturn` will return the expected array.

**Comment by tikyon** [comment]
Another workaround is to enclose it with `` in `$fieldsToReturn`

**Comment by jegelstaff** [comment]
Yeah.... since there's a workaround to pass in a clean string, this will go into the backlog. The solution will be fairly simple, just need to unpack the fieldsToReturn and encapsulate them in the method before putting them back together and using them. Just a touch too much code to do it on the spot though, mostly because of the need for testing.
